SESSION_FACILITATOR_GUIDE.md

```diff
@@ -0,0 +1,31 @@
+# 💁🏽 Session Facilitator Guide
+
+This document is intended to provide a helpful starting point or reference for running sessions during summits.
+The new learnings and tools from a summit will ideally be PR'd in when each is concluded.
+
+🎉 Lists are included 🎉
+
+## 📅 in the days leading up to the session
+- [ ] If applicable, port the agenda into [tcq](https://tcq.app/)
+- [ ] Set a chair/facilitator for the session
+
+## 🛫 pre-flight checklist (when about to begin the session)
+- [ ]  If not pre-arranged, find consensus on a chair for the session. Scheduled sessions are likely to already have a facilitator/chair. Ad-hoc sessions will need someone to make sure everyone walks out of the room with actionable items.
+- [ ] If not pre-arranged, ask the table for volunteer note taker.
```

Comment:
+1!

Reply:
Zoom accounts are a challenge.  I believe we now have more than one, but I've not had time to ask/follow up on which ones we have/how to use.  @bnb can you chime in?

Reply:
I have a zoom account that I can use for calls, but not for webinars (like the TSC and CommComm meetings).

Reply:
Please read/write here for discussion on zoom: https://github.com/nodejs/summit/pull/113#discussion_r221520539